dir.traversal.targets

```diff
@@ -112,6 +112,34 @@
     <Message Text="%(PackageIds.Identity)" Importance="High" />
   </Target>
 
+  <Target Name="GetTargetPath" DependsOnTargets="FilterProjects" Returns="@(_ProjectTargetPath)">
+    <SelectTargetPath TargetPaths="@(Project)">
+      <Output TaskParameter="TargetPath" ItemName="_TargetProject"/>
+    </SelectTargetPath>
+    <MSBuild Targets="GetTargetPath"
```

Comment:
Because some of the projects don't have a "default" build. Without this everything that doesn't build AnyOS will cause all referencing projects to fail because the reference is missing. AccessControl is the specific build that caused this.